### ISSUE TITLE
Flaring fugitives high for low production fields

### ISSUE THREAD

**Issue by qlangfitt** [arbrandt/OPGEE#157]
Flaring fugitives on the Flaring sheet (M84) are based on the component method fugitive rate for a flare. The way it's set up, every field with flaring will have a fugitive loss rate of 51 kg CH4/d (I think, since there is no formula in the flare count on the component fugitives sheet - see side note). The LCFS baseline inputs all assume an FOR of 13, so all have flaring turned on. For low production fields, the 51 kg/d fugitive rate becomes an issue as it can dominate lifecycle GHG emissions. This should have virtually no impact on the baseline crude CI since the low production fields also contritube very little to the weighted CI, but I think still worth some time to think about adjusting. Most calculations in OPGEE scale by the size of the field (production or # of wells).

Side note: Shouldn't the number of flares on the component fugitives sheet be based on the number of flares from the Flaring sheet in M62?

**Comment by smasnadi** [comment]
Yes, we realized this issue with Fugitives. Jeff is working on it to see what to do for the fields with low production rate.

**Comment by qlangfitt** [comment]
The cell in question (Flaring M84) got messed up with the retooling of the fugitives model. It now references the "Sample Sum" for liquids unloading, clearly just an oversight on adjusting it. However, "flaring" is no longer on the component fugitives process unit list.
